Script.js

```diff
@@ -12,3 +12,5 @@ document.getElementById("btn").addEventListener("click", getColor);
 //initial call
 getColor();
 console.log("hii this is test for how pull req. works.");
+
+console.log("just only for other tests.");
```

Comment:
why you insert this, because i inserted console already.

Reply:
because it's my choice

Reply:
chal gandu

Reply:
close kr isko gandu khi ka befaltu ke pulls dalta rhta hai